desktop-src/direct3dhlsl_current/hlsl-landing-page.md

```diff
@@ -0,0 +1,9 @@
+# High-level shader language (HLSL)
```

Comment:
Let's describe and link the sub-headings under this landing page in the TOC.  Let's do the same for the section pages (adding pages if necessary).  We can follow the example of the legacy HLSL landing/section pages.

The goal is to help people find the documentation they are looking for by summarizing the sections and the types of content found within them.  Point out when certain types of content divisions might not be immediately obvious.  Such as how we place certain things under the built-in section rather than the language section, when it's not immediately obvious where they might be found.